File: python/get_live_point.py
from client import create_native_client
import nativeconnect.api.v1.nativeconnect_pb2 as native

def main():
    client = create_native_client()

    request = native.GetLivePointRequest(
        vin="LRW3F7FS9PC893577"
    )

    response = client.GetLivePoint(request)

    print(response)

     
if __name__ == "__main__":
    main()

File: typescript/src/get_live_point.ts
import { createNodeClient } from "./client"
import {create, toJsonString} from "@bufbuild/protobuf";
import {
  GetLivePointRequestSchema,
  GetLivePointResponseSchema
} from "@buf/nativeconnect_api.bufbuild_es/nativeconnect/api/v1/nativeconnect_pb";

const client = createNodeClient()

const request = create(GetLivePointRequestSchema, {
  vin: "LRW3F7FS9PC893577"
})

const response = await client.getLivePoint(request)

console.log(toJsonString(GetLivePointResponseSchema, response, { prettySpaces: 2 }))
